src/mariadb.ts

```diff
@@ -0,0 +1,68 @@
+const mariadb = require("mariadb");
+const uuidv4 = require('uuid/v4');
+
+export class MariaDB {
+    constructor() {
+
+    }
+
+    public async register(username: string, password: string) {
+        const conn = await this.createConnection()
+        const otherUsers = await conn.query(
+            `SELECT * 
+             FROM Users
+             WHERE UserName="${username}"`
+        );
+        if (otherUsers.length > 0) {
+            conn.end();
+            throw new Error('Username is taken');
+        }
+        await conn.query(
+		    `INSERT INTO Users (UserName, Password, Credit) values (
+			    "${username}",
+                "${password}",
+                0
+            )`);
+        conn.end();
+        
+        return await this.loginUser(username, password);
+    }
+
+    public async loginUser(username: string, password: string) {
+        const conn = await this.createConnection();
+        const requestedUser = await conn.query(
+            `SELECT *
+             FROM Users
+             WHERE UserName="${username}"`
+        );
+        if (requestedUser.length == 0) {
+            throw new Error('Invalid Username');
+        }
+        if (requestedUser[0].Password != password) {
+            throw new Error('Invalid Password')
+        }
+        const sessionToken = uuidv4().substring(0, 16);
+        const expirationDate = new Date();
+        expirationDate.setTime(expirationDate.getTime() + (24 * 60 * 60 * 1000));
```

Comment:
Nit: maybe just add a variable to give a name to the expiration numbers. like `oneDay = 24 * 60 * 60 * 1000` (i think it's a day?) 
There's also the argument for no temporary variables haha, but with stuff like this, I feel like it's nice to know how long it is at a glance.